maintainers/maintainer-list.nix

```diff
@@ -734,6 +734,12 @@
     githubId = 1296771;
     name = "Anders Riutta";
   };
+  armijnhemel = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add armijnhemel``.

Reply:
> Please follow the contributing guide when creating your commits.

In my days there wasn't even such a guide! :-P

Kidding aside, what is the best way forward? revert that particular commit?

Reply:
```
❯ git reset --soft HEAD~6
❯ git status
HEAD detached from FETCH_HEAD
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   maintainers/maintainer-list.nix
	new file:   pkgs/development/python-modules/packageurl-python/default.nix
	modified:   pkgs/top-level/python-packages.nix
❯ git restore --staged pkgs
❯ git status
HEAD detached from FETCH_HEAD
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   maintainers/maintainer-list.nix

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkgs/top-level/python-packages.nix

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	pkgs/development/python-modules/packageurl-python/
```

And then start commit things one by one again.